projects/batfish/.project

```diff
@@ -7,16 +7,6 @@
 	<buildSpec>
 		<buildCommand>
 			<name>org.eclipse.ui.externaltools.ExternalToolBuilder</name>
```

Comment:
I don't know why you had to modify this file. I don't want to merge this change. Which version of XText are you using? Perhaps if you update to the latest this change won't be necessary.